src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
```

Comment:
BEM doesn't allow us to do so. Better stylize parent element to center calendar.

Reply:
fixed